src/subdomains/supporting/payin/strategies/send/impl/base/send.strategy.ts

```
@@ -93,8 +93,17 @@ export abstract class SendStrategy implements OnModuleInit, OnModuleDestroy {
     return this.transactionHelper.getBlockchainFee(asset, true);
   }
 
-  protected async getEstimatedFee(asset: Asset, amount: number): Promise<{ nativeFee: number; targetFee: number }> {
-    const nativeFee = await this.payoutService.estimateFee(asset, this.getForwardAddress().address, amount, asset);
+  protected async getEstimatedFee(
+    asset: Asset,
+    amount: number,
+    targetAddress?: string,
+  ): Promise<{ nativeFee: number; targetFee: number }> {
+    const nativeFee = await this.payoutService.estimateFee(
+      asset,
+      targetAddress ?? this.getForwardAddress().address,
+      amount,
```

Do we need this fallback? Maybe throw an error if no `targetAddress`?